tests/baselines/reference/user/uglify-js.log

```diff
@@ -105,6 +105,8 @@ node_modules/uglify-js/lib/propmangle.js(70,45): error TS2339: Property 'prototy
 node_modules/uglify-js/lib/propmangle.js(80,29): error TS2554: Expected 0 arguments, but got 1.
 node_modules/uglify-js/lib/propmangle.js(94,30): error TS2554: Expected 0 arguments, but got 1.
 node_modules/uglify-js/lib/propmangle.js(146,29): error TS2554: Expected 0 arguments, but got 1.
+node_modules/uglify-js/lib/scope.js(88,21): error TS2339: Property 'defun' does not exist on type 'SymbolDef'.
+node_modules/uglify-js/lib/scope.js(88,35): error TS2339: Property 'defun' does not exist on type 'SymbolDef'.
```

Comment:
leave it uglify to choose names from COMMON LISP.

I'm not sure why this wasn't an error before; defun is not assigned in the body of any SymbolDef method, only in random functions elsewhere in the module. It could be that checking of the SymbolDef.prototype object literal is correctly delayed so that it doesn't turn up `any`.